### PR TITLE
Updated command usage

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Ref: #692 

Modified the usage for the CLI commands. Added `[]` wherever necessary.

### CODE REVIEW

File: cmd/amp/config.go [hunk 1]
```diff
@@ -15,9 +15,10 @@ import (
 func init() {
 	// configCmd represents the Config command
 	configCmd := &cobra.Command{
-		Use:   "config",
+		Use:   "config [ARG..]",
```
Comment (generalhenry) [inline cmd/amp/config.go]: `[KEY] [VALUE]`

Comment (neha-viswanathan) [reply cmd/amp/config.go]: Updated

File: cmd/amp/function.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ var (
 	}
 
 	createFunctionCmd = &cobra.Command{
-		Use:   "create NAME IMAGE",
+		Use:   "create FUNC-NAME IMAGE",
 		Short: "Create a function",
 		Long: `The create command registers a function with the specified name and image.
 	If successful, a function id is returned.`,
```
Comment (JosephGJ) [inline cmd/amp/function.go]: Nit Picky. This line creates an indent on the CLI.

```
The create command registers a function with the specified name and image.
        If successful, a function id is returned.
```

**Review by JosephGJ — CHANGES_REQUESTED** [review]
Minor change. LGTM

**Review by JosephGJ — APPROVED** [review]

**Review by subfuzion — APPROVED** [review]
LGTM

**Pull request merged**